src/hotspot/cpu/aarch64/templateInterpreterGenerator_aarch64.cpp

```diff
@@ -1410,7 +1409,8 @@ address TemplateInterpreterGenerator::generate_native_entry(bool synchronized) {
     __ mov(c_rarg0, rthread);
     __ mov(rscratch2, CAST_FROM_FN_PTR(address, JavaThread::check_special_condition_for_native_trans));
     __ blr(rscratch2);
-    __ maybe_isb();
+    // An instruction sync is required here after the call into the VM. However,
+    // that will have been caught in the VM by a cross_modify_fence call.
```

Comment:
I wanted to avoid mentioning code that no longer exists. (Maybe it's best to just drop the comment?)

How about:

// When we return from the VM, the instruction stream may have
// been modified. Therefore needs an isb is required. The VM will
// have already done this by calling cross_modify_fence().